tests/backend/test_backend_reference.py

```diff
@@ -6,10 +6,10 @@
 import platform
 import sys
 import unittest
+from pathlib import Path
```

Comment:
```suggestion
import pathlib
```

Import modules only: https://google.github.io/styleguide/pyguide.html#224-decision